### DISCUSSION TITLE
关于模组管理界面的优化建议

### DISCUSSION THREAD

**Discussion by Spturate** [Meloong-Git/PCL#1453] [提问] [answered]
希望在模组管理界面能添加一个已禁用模组的按钮用来显示所有被禁用的模组，这样也方便排除bug，像几百个模组的还忘记名字了在这里找来找去还是有点麻烦的
<img width="738" alt="屏幕截图 2023-03-16 163129" src="https://user-images.githubusercontent.com/30800649/225560114-88ff4b5b-78a6-4986-93e8-adb227dd9985.png">

**Comment by Aether-254** [comment]
建议提交到 “优化建议”

**Comment by WForst-Breeze** [comment, marked as answer]
如有优化建议，请提交 Issues，感谢您的支持～
